test/e2e/setup_test.go

```diff
@@ -88,6 +89,13 @@ var (
 	ignoreConditionLTTReasonAndMessageFields = cmpopts.IgnoreFields(metav1.Condition{}, "LastTransitionTime", "Reason", "Message")
 	ignoreAgentStatusHeartbeatField          = cmpopts.IgnoreFields(clusterv1beta1.AgentStatus{}, "LastReceivedHeartbeat")
 	ignoreNamespaceStatusField               = cmpopts.IgnoreFields(corev1.Namespace{}, "Status")
+
+	crpStatusCmpOptions = cmp.Options{
+		cmpopts.SortSlices(lessFuncCondition),
```

Comment:
where were these function used before? wouldn't golang complain about it if there is not used before?

Reply:
they're used in https://github.com/Azure/fleet/blob/main/test/e2e/actuals_test.go#L105.

The IDE will give the hints.